sporks.json

```diff
@@ -1,950 +1,995 @@
+              "execution-state-wal-chunks": ""
+            },
+            "s3": {
+              "root-checkpoint-file": "",
+              "root-protocol-state-snapshot": "",
+              "node-info": "",
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "v0.10.1",
+          "flow-dps-tag": "",
+          "cadence-tag": "v0.9.2",
+          "docker-tag": "v0.10.1",
+          "access-node": "access-001.mainnet1.nodes.onflow.org:9000"
+        },
+        {
+          "id": 0.9,
+          "name": "Candidate-9",
+          "spork-date": "2020-09-25T16:00:00Z",
+          "root-height": "6483246",
+          "root-parent-id": "9131733835702b0d6321088bddb4642a4964bb5c630440ccb0de47bdbc371d1a",
+          "root-state-commit": "90c6f406f5d21880d525ad4702cb249509b85e7f745db2de67e9fe541a56da4c",
+          "git-commit-hash": "",
+          "state-artefacts" : {
+            "gcp": {
+              "root-checkpoint-file": "https://storage.googleapis.com/flow-genesis-bootstrap/candidate-9-execution/public-root-information/root.checkpoint",
+              "root-protocol-state-snapshot": "",
+              "node-info": "https://storage.googleapis.com/flow-genesis-bootstrap/candidate-9-execution/public-root-information/node-infos.pub.json",
+              "execution-state-wal-chunks": ""
+            },
+            "s3": {
+              "root-checkpoint-file": "",
+              "root-protocol-state-snapshot": "",
+              "node-info": "",
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "v0.9.3",
+          "flow-dps-tag": "",
+          "cadence-tag": "v0.8.2",
+          "docker-tag": "v0.9.3",
+          "access-node": ""
+        },
+        {
+          "id": 0.8,
+          "name": "Candidate-8",
+          "spork-date": "2020-09-08T16:00:00Z",
+          "root-height": "4972987",
+          "root-parent-id": "5bc2b0900a5138e39d9209a8fe32e14b3e5c884bd36d2a645620f746b7c8bd47",
+          "root-state-commit": "6b9161a225b087a461ec95e710fdf4e73f6d6c9401ebf066207a021dced4ce5e",
+          "git-commit-hash": "",
+          "state-artefacts" : {
+            "gcp": {
+              "root-checkpoint-file": "https://storage.googleapis.com/flow-genesis-bootstrap/candidate-8-execution/public-root-information/root.checkpoint",
+              "root-protocol-state-snapshot": "",
+              "node-info": "https://storage.googleapis.com/flow-genesis-bootstrap/candidate-8-execution/public-root-information/node-infos.pub.json",
+              "execution-state-wal-chunks": ""
+            },
+            "s3": {
+              "root-checkpoint-file": "",
+              "root-protocol-state-snapshot": "",
+              "node-info": "",
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "",
```

Comment:
and these as well

Reply:
Same, again - there's nothing noted in the Sporks notion doc and nor in the current public docs for past sporks. Any suggestions you have to identify the missing data would be great